docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -1304,20 +1023,20 @@
       "text/plain": [
        "             dmm_v1\n",
        "dac_ch1            \n",
-       "0.000000   5.032144\n",
-       "2.777778   2.840059\n",
-       "5.555556   1.726878\n",
-       "8.333333   1.005160\n",
-       "11.111111  0.580161"
+       "0.000000   5.128481\n",
+       "2.777778   2.812913\n",
+       "5.555556   1.685043\n",
+       "8.333333   0.883477\n",
+       "11.111111  0.560485"
       ]
      },
-     "execution_count": 31,
+     "execution_count": 24,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "df = dataset.to_pandas_dataframe_dict()['dmm_v1']\n",
+    "df = dataset_1d.to_pandas_dataframe_dict()['dmm_v1']\n",
```

Comment:
similar I would use the to_xarray_dataarray method

Reply:
done